event-sourcing-redis/Readme.md

```diff
@@ -0,0 +1,141 @@
+# Event Sourcing with Redis
+It is a common practice to use asynchronous messaging technologies to implement communication between multiple microservices.
+It helps to have scalable, fault tolerant and decoupled architecture. Kafka is go-to solution when it comes to streaming pipelines and publish/subscribe systems.
```

Comment:
```suggestion
It provides scalable, fault tolerant and decoupled architecture. Kafka is go-to solution when it comes to streaming pipelines and publish/subscribe systems.
```

Reply:
If I understood the idea, this is more appropriate and leaves no place for misinterpretation.